CandidateReranking.md

```diff
@@ -32,23 +31,22 @@ OAUTH credentials are **required**.
 A request is composed of 4 main parts:
 `reranker_config`, `source`, `query`, and `profiles`.
 
-#### Top Level
 | Param    | Type | Description
 |----------|------|--------|
 | reranker_config | String | **Required**: Accepted Value: **RERANKER_V1** The `reranker_config` determines which learned model to apply to your incoming rerank request. Currently there is only one reranker model to call "RERANKER_V1".
 | source | String | **Required**: Accepted Values: **EDGE** or **MY_SUPPLY**. The source identifies where profile data was attained. A model can be learned based on the source providing more accurate results per each data format.
-| query | [Query](#query-object-parts) | **Required**: The original fully enriched query sent to SOLR for the list of Candidate Profiles.
-| profiles | [Profile[]](#profile-and-objec-parts) | **Required**: Array of profiles to be reranked.
+| query | [Query](#query) | **Required**: The original fully enriched query sent to SOLR for the list of Candidate Profiles.
```

Comment:
comma after "original"

"fully enriched" should *maybe* be hyphenated? (ponder)

"Candidate Profiles" should be downcased